backend/custom_plugins/models.py

```diff
@@ -14,6 +14,15 @@ def __str__(self):
         return self.name
 
 class Blurb(CMSPlugin):
+    TYPE_CHOICES = (
+        ('link', 'Link'),
+        ('button', 'Button'),
+    )
+    type = models.CharField(
+        max_length=8,
+        choices = TYPE_CHOICES,
```

Comment:
Try to be consistent with whitespace (e.g. not use `foo = bar` on some lines, and `foo=bar` on others).